gptel-transient.el

```diff
@@ -184,6 +184,26 @@ which see."
         (propertize display-if-true
                     'face (if value 'transient-value 'transient-inactive-value))))))
 
+(defclass gptel--multi-switches (transient-lisp-variable)
```

Comment:
This class will not be required, see below comment on `gptel--infix-context-destination`.